app/views/embed/body/_file.html.erb

```diff
@@ -0,0 +1,83 @@
+<div class='sul-embed-body sul-embed-file'
+      data-sul-embed-theme="<%= viewer.asset_url('file.css') %>"
```

Comment:
Sounds good for another PR. There's already a lot of moving stuff in this one. I'm trying not to change more than necessary